Core/StitchCoreSDK/Sources/StitchCoreSDK/StitchError.swift

```diff
@@ -24,8 +24,18 @@ public enum StitchError: Error {
      * thrown when attempting to decode the response. An error code is included, which indicates whether the error
      * was a transport error or decoding error.
      */
-    case requestError(withError: Error, withRequestErrorCode: StitchRequestErrorCode)
+    case requestError(withMessage: String, withRequestErrorCode: StitchRequestErrorCode)
```

Comment:
[neat] I did not know you could have multiple enum cases with the same name, but different types of associated values.

Reply:
👍